haven/haven_utils/__init__.py

```diff
@@ -346,6 +346,20 @@ def copy_code(src_path, dst_path, verbose=1):
     # Create destination folder
     os.makedirs(dst_path, exist_ok=True)
 
+    # Define the command for copying the code using rsync
+    if os.path.exists(os.path.join(src_path, ".havenignore")):
+        rsync_code = (
+            "rsync -av -r -q  --delete-before --exclude='.*'  --exclude-from=%s \
+                        --exclude '__pycache__/' %s %s"
+            % (os.path.join(src_path, ".havenignore"), src_path, dst_path)
+        )
+    else:
+        rsync_code = (
+            "rsync -av -r -q  --delete-before --exclude='.*' \
+                        --exclude '__pycache__/' %s %s"
+            % (src_path, dst_path)
+        )
+
     # Define the command for copying the code using rsync
     rsync_code = (
```

Comment:
should we remove this line? It looks like it overrides `rsync_code`

Reply:
Fixed!